test/test.dart

```diff
@@ -388,27 +388,22 @@ void xtest(String? name, dynamic Function() testFunction, {dynamic skip, Map<Str
 }
 
 BaasHelper? baasHelper;
-Map<String, String?> _testArgs = {};
-
-Future<void> setupTests(List<String>? args) async {
-  _testArgs = parseTestArguments(args);
-  testName = _testArgs["name"];
 
+void setupTests() {
   setUpAll(() async {
-    baasHelper = await BaasHelper.setupBaas(_testArgs);
-  });
+    baasHelper = await BaasHelper.setupBaas();
 
-  setUp(() {
     Realm.logger = Logger.detached('test run')
       ..level = Level.ALL
       ..onRecord.listen((record) {
-        if (record.level.value >= RealmLogLevel.warn.value) {
-          print('${record.time} ${record.level.name}: ${record.message}');
-        }
-
```

Comment:
Is this intentional? I guess there's still value in printing warnings/errors, even if the test succeeds.

Reply:
It is intentional, but perhaps a bit opinionated :-). If the test fails, it will be included in the trace regardless. I'm okay with adding it back, if you prefer.